src/Cards.py

```diff
@@ -15,6 +15,12 @@ def __init__(self, colour, cardType, cardNumber):
         super().__init__(colour, cardType)
         self.cardType = 'regular'
         self.cardNumber = cardNumber
+    
+    def build(self, surface):
+        draw.rect(surface,self.colour,(100,300,100,300))
+    
+    #super().build(self, surface):
+    #^ not sure if inheritance can be done this way lol
```

Comment:
Can. Just checking, what you do you plan for calling the parent build function?

Reply:
honestly not sure yet... when i wrote these i realised it wasn't exactly what i envisioned...